tests/.pest/snapshots/Countries/BrazilTest/it_can_calculate_brazil_holidays.snap

```diff
@@ -0,0 +1,50 @@
+[
+    {
+        "name": "Dia de Ano Novo",
+        "date": "2024-01-01"
+    },
+    {
+        "name": "Carnaval",
+        "date": "2024-02-13"
+    },
+    {
+        "name": "Sexta-feira Santa",
+        "date": "2024-03-29"
+    },
+    {
+        "name": "P\u00e1scoa",
```

Comment:
Is this an official one? It's not on the list: https://publicholidays.com.br/2024-dates/

Reply:
Removed.